### ISSUE TITLE
Make skipping TLS verification optional

### ISSUE THREAD

**Issue by markszabo** [kovetskiy/mark#99]
Currently the underlying library, [https://github.com/kovetskiy/gopencils](https://github.com/kovetskiy/gopencils), skips TLS verification:

```
// Skip verify by default?
tr := &http.Transport{
	TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
}
```
(ref: https://github.com/kovetskiy/gopencils/blob/master/api.go#L67)

I'd like to make this a config option (e.g a flag similar to curl's ` --insecure`), to at least have the option to check the certificate (to prevent someone from snooping on the traffic and getting the confluence credentials). 

I see the work as the following:

1. Make the change in gopencils, send PR
2. Once PR is merged, release new version of gopencils
3. Make change to mark using the new version of gopencils, send PR

I'm willing to do the implementation part. Does this sounds reasonable? 

What should be the default behavior for mark? I think defaulting to checking the certificate (like curl) would be ideal, but that might break existing workflows.

**Comment by kovetskiy** [comment]
Hi @markszabo, the workflow sounds reasonable. I actually didn't expect this flag InsecureSkipVerify to be true in gopencils. Yikes.

It should be false by default, we can also add  the `--insecure-skip-tls-verify` flag, but it's not mandatory from my point of view.

Thank you for pointing out!

**Comment by markszabo** [comment]
Well, I don't need the `--insecure-skip-tls-verify` flag, so if you're okay with it, it would be way easier to just set `InsecureSkipVerify` to false in gopencils, and if someone needs it, then it can be added later. What do you think?

**Comment by kovetskiy** [comment]
Sounds good for me

**Comment by markszabo** [comment]
Friendly ping on https://github.com/kovetskiy/gopencils/pull/1 is there anything else you want me to do on this PR? 🙂

**Comment by kovetskiy** [comment]
Hi, I've just merged that PR and upgraded it as a dependency in mark, and released version 6.0. Thanks for the contribution!

**Comment by markszabo** [comment]
Awesome, thanks!
